### PR TITLE
Add server instructions with IPv6 address information

### PR DESCRIPTION
**Pull request by sunithasomanna**
- Add BMC server (with IPv6 address and update the section accordingly with sample request, response and parameters description in the API readme.
- Updated IPv4 hostname with public IP.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2366,13 +2366,11 @@ curl -i -X POST \
...
 ```
 
-
-
 >**Sample request body**
 
 ```
 {
-   "HostName":"10.24.0.4",
+   "HostName":"192.168.256.256",
```
Comment (amar-shalgar) [inline docs/README.md]: 255 is the maximum allowed number. Please change this

File: docs/README.md [hunk 2]
```diff
@@ -2439,17 +2454,34 @@ x-frame-options":"sameorigin"
...
 
+>**Sample response body (HTTP 201 status)**
+```
+ {
+   "@odata.type":"#AggregationSource.v1_1_0.AggregationSource",
+   "@odata.id":"/redfish/v1/AggregationService/AggregationSources/26562c7b-060b-4fd8-977e-94b1a535f3fb",
+   "@odata.context":"/redfish/v1/$metadata#AggregationSource.AggregationSource",
+   "Id":"26562c7b-060b-4fd8-977e-94b1a535f3fb",
+   "Name":"Aggregation Source",
+   "HostName":"192.168.256.256",
```
Comment (amar-shalgar) [inline docs/README.md]: Add a valid IP address

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 2 above)
Comment (sunithasomanna) [reply docs/README.md]: done

File: docs/README.md (hunk 1 above)
Comment (sunithasomanna) [reply docs/README.md]: done

**Review by jeevan-kamkar — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**